app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -99,11 +101,13 @@ protected void onCreate(Bundle savedInstanceState) {
         studentImageCollectionEventDao = daoSession.getStudentImageCollectionEventDao();
         studentImageDao = daoSession.getStudentImageDao();
 
-        collectionEventId = deviceId + String.format("%016d",studentImageCollectionEventDao.count() + 1);
+        // Create Collection Even for Student Images
+        collectionEventId = String.format("%06d",studentImageCollectionEventDao.count() + 1);
```

Comment:
Since this will be objects generated on the tablets instead of downloaded from the database, the id can be auto-generated. No need to set it manually. I'll upgrade the DAO code to make this easier.